tests/t0002-actions.sh

```diff
@@ -16,6 +16,13 @@ echo "TODO: foo"
 EOF
 chmod +x foo
 
+cat > foo2 << 'EOF'
+shift
+IFS=-
```

Comment:
Why did you put dashes between the output?

Reply:
To be sure that the individual arguments are passed separately, and no wrong splitting on contained spaces occurs. I've added a comment to document this.